gpt_engineer/core/ai.py

```diff
@@ -463,69 +463,63 @@ def num_tokens_from_messages(self, messages: List[Message]) -> int:
         n_tokens += 2  # every reply is primed with <im_start>assistant
         return n_tokens
 
+    def _check_model_acess_and_fallback(self):
```

Comment:
It should be access right?
Apart from that, happy to merge

Reply:
yep done